doc/source/sdk/v2/zaf_v2.md

```diff
@@ -0,0 +1,146 @@
+## Zendesk App Framework v2 Beta
+The Zendesk App Framework (ZAF) v2 introduces a new way of building Zendesk apps using iframes.
+
+The new framework allows you to build powerful apps using whatever technologies you are used to along with all the capabilities you expect from the existing framework.
+
+### How does it work?
+ZAF v2 apps include a manifest.json file and an assets folder. The manifest file includes a slightly modified locations property, which specifies URL paths for each location. These URL paths can be absolute, pointing to an external page, or relative, pointing to a html file inside the assets folder.
+
+e.g.
+```json
+"location": {
+  "zendesk": {
+    "ticket_sidebar":       "assets/iframe.html?location=ticket_sidebar",
+    "new_ticket_sidebar":   "assets/iframe.html?location=new_ticket_sidebar",
+    "user_sidebar":         "assets/user_sidebar.html",
+    "organization_sidebar": "assets/organization_sidebar.html",
+    "nav_bar":              "https://dashboard.myapp.com/nav_bar",
+    "top_bar":              "https://dashboard.myapp.com/top_bar"
+  }
+}
+```
+
+The HTML files referenced on the manifest file will need to include the ZAF SDK v2 library in a script tag. This will allow your iframe to interact with the new ZAF SDK APIs documented below.
+
+i.e.
+```html
+<script src="http://assets.zendesk.com/apps/sdk/2.0.0-beta1/zaf_sdk.js"></script>
+```
+
+### Feature parity
+While ZAF v2 aims to support all existing framework features, in some cases due to the nature of the new architecture some features may be missing or incomplete. For more information check out our feature parity status page [here]().
+
+### Migration
+Existing Zendesk Apps can migrate to ZAF v2 using the Iframe App Template. Check out our [Migration Guide]() for more information.
+
+### Creating a new ZAF v2 app
+The Zendesk App Tools (ZAT) provides a new option to create ZAF v2 apps, also called Iframe Only apps.
+
+```
+$ zat new --iframe-only
+```
+
+After following the usual prompts ZAT will ask you if you want to enter an iframe URI or leave it blank to use a default local template page. If you enter an iframe URI ZAT will copy it to the ticket_sidebar location on the manifest file. If you choose to leave it blank ZAT will create a HTML template and copy its relative path to the ticket_sidebar location on the manifest file.
+
+## ZAF SDK v2
+ZAF SDK v2 allows you to interact with the framework directly from your iframe. If you haven't used ZAF SDK before check out [Iframes In Apps: ZAFClient API](./iframes_in_apps#zafclient-api), just keep in mind the `postMessage` API described there has been deprecated in v2, since ZAF v2 apps interact directly with the framework, rather than through a legacy Zendesk app.
```

Comment:
> If you haven't used ZAF SDK before check out [Iframes In Apps: ZAFClient API](./iframes_in_apps#zafclient-api). **Keep in mind the `postMessage` API described there has been deprecated in v2**.  ZAF v2 apps interact directly with the framework, rather than through a legacy Zendesk app.

Also, worth describing *how* it will be easier/simpler with v2 now that the iframe app interacts directly?